### ISSUE TITLE
Using the "com" identifier in the e-mail seems unfriendly.

### ISSUE THREAD

**Issue by dahenson** [elementary/houston#451]
![screenshot from 2017-05-18 11 53 53](https://cloud.githubusercontent.com/assets/2007067/26214135/e4c72e86-3bc0-11e7-93b5-a407f777e95c.png)
![screenshot from 2017-05-18 11 56 16](https://cloud.githubusercontent.com/assets/2007067/26214177/05aee530-3bc1-11e7-9d38-1061c27e3d07.png)

I'm not a huge fan of using "com.github.needle-and-thread.vocal" in both the title and the body of the purchase e-mail. Is there any way to just say "Congrats on installing Vocal?" I think that would be more friendly for customers.

**Comment by btkostner** [comment]
It's a known issue. That is how it was originaly in the mockup, but we currently do not store the desktop name.

**Comment by MichaelRupertDev** [comment]
Working on this, close to done. 

Since we currently have no "desktop" name stored, should we use the name of the repo?

**Comment by MichaelRupertDev** [comment]
With that PR merged, what else needs to be done for this issue?

**Comment by btkostner** [comment]
I'm not going to close this issue because we still want to use the appstream name that the developer has wrote. The hard part is that we will need to update the database with the appstream information when we test.

**Comment by MichaelRupertDev** [comment]
What more work needs to be done to complete this bounty?

**Comment by btkostner** [comment]
We need to save the `.appdata.xml` file data to the database. This way we can use the actual name the developer gives the application in the email. The test file for `appdata` is [here](https://github.com/elementary/houston/blob/master/src/flightcheck/pipes/AppData/index.js). It's a mess. I'm sorry.

**Comment by MichaelRupertDev** [comment]
I'll take a look at this when I have some time

**Comment by btkostner** [comment]
Hop in gitter if you need any help understanding how all of this glues together

**Comment by rupertdev** [comment]
I totally forgot I ever worked on this, interested in finishing it up, does your comment above still hold @btkostner ?

**Comment by btkostner** [comment]
Yep. I might not respond right away, but I try to check it every day
